Server/Content/Sources/App/Models/Text.swift

```diff
@@ -0,0 +1,27 @@
+import Fluent
+import Vapor
+
+final class Text: Model, Content {
+    static let schema = "texts"
+    
+    @ID(key: .id)
+    var id: UUID?
+
+    @Field(key: "title")
+    var title: String
+    
+    @Field(key: "text")
+    var text: String
+    
+    @Field(key: "backward_text")
+    var backwardText: String?
```

Comment:
Are you really sure that everyone is going to need backwardsText? For example Blender game needs shuffled text, but not the backwards.